src/Domain/TagsTimeDependent.hpp

```diff
@@ -27,25 +27,32 @@ namespace domain {
 /// \ingroup ComputationalDomainGroup
 /// \brief %Tags for the domain.
 namespace Tags {
-/// The Inertial coordinates, Inertial grid velocity, and the Jacobian and
-/// inverse Jacobian from the Grid to the Inertial frames.
+/// The Inertial coordinates, the inverse Jacobian from the Grid to the Inertial
+/// frame, the Jacobian from the Grid to the Inertial frame, and the Inertial
+/// mesh velocity.
+///
+/// The type is a `boost::optional`, which, when is not valid, signals that the
+/// mesh is not moving. Thus,
+/// `static_cast<bool>(coordinates_velocity_and_jacobian)` can be used to check
+/// if the mesh is moving.
```

Comment:
You could add another compute tag `IsMeshMoving` or similar that does what you describe here. Just a suggestion, and doesn't need to happen in this PR

Reply:
Yes, but that is not helpful.

Reply:
what is not helpful?

Reply:
Having another tag. You always need to be able to get the velocity, and you always need to get it since it's a runtime overload, so having another tag to check just adds more compute tags to the DataBox without any benefit